src/sql/workbench/browser/modelComponents/loadingComponent.component.ts

```diff
@@ -13,13 +13,11 @@ import * as azdata from 'azdata';
 import { ComponentBase } from 'sql/workbench/browser/modelComponents/componentBase';
 import { localize } from 'vs/nls';
 import { IComponent, IComponentDescriptor, IModelStore } from 'sql/platform/dashboard/browser/interfaces';
+import { status } from 'vs/base/browser/ui/aria/aria';
 
 @Component({
 	selector: 'modelview-loadingComponent',
 	template: `
-		<div role="status" aria-live="polite" class="modelview-loading-component-status-message">
```

Comment:
glad that you fixed this, i also realized this problem but didn't get a chance to look into it. looked into the vscode status implementation and it looks good.

Reply:
Note that this isn't perfect. If multiple things send off a status message at the same time then it'll usually only announce one of them. There isn't really a good way to solve this though that I can think of since there's no way to detect if something has actually been announced. Might be something we have to come up with a solution for later if we start needing multiple messages to be announced in a row.